script/check-aws-cdk

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
+
+set -e
+
+DIR=$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )
+
+LIBRARY_TO_CHECK="@aws-cdk/core"
+TOLERANCE=5
+
+VERSIONS=$(npm view $LIBRARY_TO_CHECK versions --json)
+
+TOTAL_VERSIONS=$(echo "$VERSIONS" | jq '. | length')
+
+INSTALLED_VERSION_NUMBER=$(jq -r ".dependencies.\"${LIBRARY_TO_CHECK}\"" < "$DIR/../package.json")
+
+INSTALLED_VERSION_POSITION=$(echo "$VERSIONS" | jq ". | index(\"${INSTALLED_VERSION_NUMBER}\")")
+
+DIFFERENCE=$((TOTAL_VERSIONS - INSTALLED_VERSION_POSITION))
+
+if [ $DIFFERENCE -gt $TOLERANCE ]; then
+  echo "⚠️  This repository is currently using version $LIBRARY_TO_CHECK at version $INSTALLED_VERSION_NUMBER."
```

Comment:
```suggestion
  echo "⚠️  This repository is currently using $LIBRARY_TO_CHECK at version $INSTALLED_VERSION_NUMBER."
```

Reply:
Ha, just noticed that too and force pushed.